WasmAllocModule/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "alloc"
+version = "0.1.0"
+authors = ["Mohammad-nassar10 <mohammad.nassar@ibm.com>"]
+edition = "2018"
+#build = "build.rs"
```

Comment:
Remove all commented out lines in all files (keep it somewhere in history in case we need it later)